### PR TITLE
Upgrade instructions for model names that contain spaces

### LINKED ISSUE #5347: [Core] Upgrade guide for model names that contain spaces
### Link to the page(s) on docs.getdbt.com requiring updates

Migration guide for v1.8

### Tell us more about this update

For folks that have a model name that contains spaces, provide instructions for how to upgrade via [custom aliases](https://docs.getdbt.com/docs/build/custom-aliases).

#### Before

`models/my model.sql`

```sql
select 1 as id
```

`models/_models.yml`

```
models:
  - name: "my model"
```

#### After - option 1

`models/my_model.sql`

```sql
{{ config(alias="my model") }}

select 1 as id
```

`models/_models.yml`

```
models:
  - name: "my model"
```

#### After - option 2

`models/my_model.sql`

```sql
select 1 as id
```

`models/_models.yml`

```
models:
  - name: my_model
    config:
      alias: "my model"
```

### Reviewers/Stakeholders/SMEs

.

### Related GitHub issues

- https://github.com/dbt-labs/dbt-core/issues/9397
- https://github.com/dbt-labs/dbt-core/pull/9886
- https://github.com/dbt-labs/docs.getdbt.com/issues/5275
- https://github.com/dbt-labs/docs.getdbt.com/pull/5295

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by dbeatty10**
[Preview](https://docs-getdbt-com-git-dbeatty10-patch-1-dbt-labs.vercel.app/docs/dbt-versions/core-upgrade/upgrading-to-v1.8#spaces-in-dbt-model-names)

## What are you changing in this pull request and why?
resolves #5347

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

### CODE REVIEW

File: website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md [hunk 1]
```diff
@@ -88,6 +88,8 @@ We will begin deprecating support for spaces in dbt model names in v1.8 (raising
 - Spaces in a model name make it impossible to `--select` the model name because the argument gets split into pieces over spaces very early in the pipeline.
 - Most warehouses do not accept a table, or other object, with a space in its name.
 
+To upgrade, replace any spaces in the model file name with an underscore and update any associated YAML that contains the model name to match. You can keep spaces in the database table name by configuring a [custom `alias`](docs/build/custom-aliases#usage).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md]:
```suggestion
To upgrade, replace any spaces in the model file name with an underscore and update any associated YAML that contains the model name to match. You can keep spaces in the database table name by configuring a [custom `alias`](/docs/build/custom-aliases#usage).
```

**Review by mirnawong1 — APPROVED** [review]
🚢

**Comment by dbeatty10** [comment]
Thank you for fixing and shipping this @mirnawong1 !

I goofed the hyperlink like three different times, but you saved the day 🏆

**Comment by mirnawong1** [comment]
you got it, @dbeatty10 !!

**Pull request merged**